### DISCUSSION TITLE
Is it possible to get the Notification widget expanded by default

### DISCUSSION THREAD

**Discussion by Moostek** [breezy-weather/breezy-weather#1275] [Q&A] [answered]
Hello, is it possible to set up the notification widget expanded, to see the day/week forecast by the default? I would like to see the week forecast at a first glance. Now I have to expand the notification widget to get the overview. 
Thank you.

**Comment by papjul** [comment, marked as answer]
Android stacks notifications when there are many. I don't think you can force expanding notifications, otherwise every app would do that and that would be a mess. Maybe some custom ROM provides preferences to force expanding?

**Reply by Moostek** [reply]
Well, I use also another weather app and the day/week overview is firm there, all the time, no matter how many notifications I get. So I am sure that this is technically possible. It will be nice to have this feature in BreezyWeather Settings.

**Reply by papjul** [reply]
Please share the link to the source code of this app

**Comment by Moostek** [comment]
This app is not on GitHub.

**Reply by papjul** [reply]
Another website is fine

**Reply by Moostek** [reply]
It is Weawow weather app, respecting privacy too.
